openbas-api/src/main/java/io/openbas/rest/exercise/ExerciseApi.java

```diff
@@ -332,18 +333,23 @@ public Exercise removeExerciseTeamPlayers(@PathVariable String exerciseId, @Path
   @PostMapping("/api/exercises")
   @Transactional(rollbackOn = Exception.class)
   public Exercise createExercise(@Valid @RequestBody ExerciseCreateInput input) {
-    Exercise exercise = new Exercise();
-    exercise.setUpdateAttributes(input);
-    exercise.setTags(iterableToSet(tagRepository.findAllById(input.getTagIds())));
-    if (imapEnabled) {
-      exercise.setFrom(imapUsername);
-      exercise.setReplyTos(List.of(imapUsername));
-    } else {
-      exercise.setFrom(openBASConfig.getDefaultMailer());
-      exercise.setReplyTos(List.of(openBASConfig.getDefaultReplyTo()));
+    if (input != null ) {
+      if (StringUtils.isNotBlank(input.getId()))
```

Comment:
I don't get why you decided not to have dedicated endpoints. 
I think that add confusion to what the create endpoint does between creation and duplication.

Reply:
In my opinion we don't need a new endpoint for a creation, because it is.
We can discuss more about that later if you want.

Reply:
I still disagree with that. If it were the same than a creation, there would be part of the code in common between the two. In my opinion, that choice breaks the single responsability principle.
How about a POST endpoint on /api/exercises/{exerciseId} ? I think that would be RESTful and not break the SRP. Please let me know if you want to discuss further the issue.

Reply:
I change it in your way. You can check again